scripts/create_aad_assets.sh

```diff
@@ -23,7 +23,7 @@ fi
 ./scripts/aad/aad-app-reg.sh \
   --name "${TRE_ID}" \
   --swaggerui-redirecturl "https://${TRE_ID}.${LOCATION}.cloudapp.azure.com/api/docs/oauth2-redirect" \
-  --admin-consent --automation-account "${api_app_can_create_other_applications}"
+  --admin-consent --automation-account ${api_app_can_create_other_applications}
```

Comment:
Does the linter not complain on this? Do you need to ignore the linter explicitly here?

Reply:
The linter *may* complain, but those quotes generate errors in some situations (I hit this recently when deploying)

Reply:
Looks like it doesn't fire on shellcheck - but have added a comment above